src/app/shared/components/slider/slider.component.ts

```diff
@@ -19,6 +20,7 @@ export class SliderComponent implements OnInit, ControlValueAccessor {
   @Input() public min: number;
   @Input() public max: number;
   @Input() public units: string;
+  @Input() public diskOffering: DiskOffering;
```

Comment:
remove this. A slider is an independent component. use ngIf in the parent template no to show this one